core/src/test/java/com/mapzen/android/search/MapzenSearchTest.java

```diff
@@ -18,16 +21,28 @@
 import retrofit2.Response;
 
 public class MapzenSearchTest {
+  private Context context;
+  private Pelias pelias;
+  private MapzenSearch search;
+
+  @Before public void setUp() throws Exception {
```

Comment:
This naming convention is actually a legacy of JUnit 3 when the `@Before` annotation did not exist and the pre-test method had to be named `setUp()` with camel case in order to be picked up by the test runner.

Fun fact is before JUnit annotations existed all test methods had to start with the word `test` too. For example `testNotNull()`.

While starting each test method name with the word `test` has been largely abandoned in JUnit 4 now that we have the `@Test` annotation most developers still adhere to the `@Before setUp()` and `@After tearDown()` naming conventions.